code/game/gamemodes/miniantags/abduction/abductee_objectives.dm

```diff
@@ -148,3 +148,71 @@
 
 /datum/objective/abductee/sixthsense
 	explanation_text = "You died back there and went to heaven... or is it hell? No one here seems to know they're dead. Convince them, and maybe you can escape this limbo."
+
+/datum/objective/abductee/forbiddennumber
+	explanation_text = "Ignore anything in a set number of"
+
+/datum/objective/abductee/forbiddennumber/New()
+	..()
+	var/number = pick(list("two", "three", "four", "five", "six", "seven", "eight", "nine", "ten"))
+	explanation_text +=" [number], they don't exist."
+
+/datum/objective/abductee/buddy
+	explanation_text = "Being alone and in large groups are both frightening.  Try to be alone with only one other person whenever possible."
+
+/datum/objective/abductee/finality
+	explanation_text = "Death should be final and modern medicine disrupts the natural order. Don't allow anyone to be revived."
+
+/datum/objective/abductee/mispronounced
+	explanation_text = "No matter how they say it, other people keep mispronouncing your name.  Be sure to correct them whenever possible."
+
+/datum/objective/abductee/bald
+	explanation_text = "There are alien parasites masquerading as people's hair.  Save people from this invasion."
+
+/datum/objective/abductee/one
+	explanation_text = "There is only one other person in existence, he is just really good at pretending to be multiple people."
+
+/datum/objective/abductee/execution
+	explanation_text = "Try to get formally executed for a crime you didn't commit, without a false confession."
```

Comment:
> This one is just ban bait

The point of that one is execution for a crime you didnt commit, you are not allowed to commit said crime. If you do commit a crime for the objective you arent following it, its of a similar nature of the drug objective which only harms the one who got it.

Reply:
It still seems very disruptive to any security operations going on.
Also, don't forget that wrongful executions are murder still, so this can get officers into perma.

Reply:
Changed it to a outlaw objective, you are only phisically allowed to commit minor crimes without being spotted by security.